task1-gradient-descent/GradientDescent_Ferhat.ipynb

```diff
@@ -0,0 +1,1626 @@
+{
```

Comment:
<p>It's good to go deep and calculate all math in your code in the early stages to understand the theory. In practice, you are not gonna calculate this way. You find a function do it for your like here:</p><p> </p><p>https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.StandardScaler.html</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bogaziciai/assignment/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>